lib/ndr_import/table.rb

```diff
@@ -79,6 +80,15 @@ def process_line(line, &block)
       @notifier.try(:processed, @row_index)
     end
 
+    # Update 'column' values expressed as a regular expression
+    def mutate_regexp_columns(line)
+      @columns.each_with_index do |column, index|
+        next unless column['column'].is_a? Regexp
+
+        column['column'] = line[index] if column['column'] =~ line[index]
```

Comment:
does this one better off as `match?`

Reply:
Ah yep, I'd forgotten about `match?`. I had considered `match`, but it's slower than `=~`, arnd we're not doing anything with the matched value; just used it as a conditional.


```
=~ is typically used in conditionals, when you only care if something matches:

do_stuff if "foobar" =~ /foo/
do_stuff if "foobar".match(/foo/) # same effect, but probably slower and harder to read

match is typically used when you want details about what was matched:

 name = "name:bob".match(/^name:(\w+)$/)[1]
 puts name #=> 'bob'
```